core/arch/arm/plat-ls/conf.mk

```diff
@@ -5,6 +5,7 @@ $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
 $(call force,CFG_GIC,y)
 $(call force,CFG_16550_UART,y)
 $(call force,CFG_PM_STUBS,y)
+$(call force,CFG_SECONDARY_INIT_CNTFRQ,y)
```

Comment:
I think this CNTFRQ init for secondary cores is required for ARMv7 based platforms only in OP-TEE. As for ARMv8 based platforms it is already done in ATF. So we only need this macro for ls1021atwr and ls1021aqds platforms.

Reply:
Good catch, I have moved the configuration directive next to `CFG_ARM32_core`.